src/pages/stacks-blockchain/overview.md

```diff
@@ -31,17 +31,17 @@ You can view accounts, blocks, transactions, and smart contracts broadcasted to
 [@page-reference | grid-small]
 | /stacks-blockchain/managing-accounts, /stacks-blockchain/sending-tokens, /stacks-blockchain/running-testnet-node
 
-## Capabilities
+## Features
 
 ### Clarity Smart Contracts
 
-A [Smart Contract](https://academy.binance.com/glossary/smart-contract) is code running on the Stacks Blockchain that executes autonomously.
+A [Smart Contract](https://academy.binance.com/glossary/smart-contract) is simply a program that runs on a blockchain.
 
-Smart contracts on the Stacks 2.0 blockchain are written in a new language called Clarity. This language gives developers a safe way to build complex smart contracts. It's main features are:
+Smart contracts on the Stacks 2.0 blockchain are written in a new programming language called **Clarity**. Clarity is distinct from other languages designed for writing smart contracts in a few ways:
 
-- **Predictability**: The Clarity language uses precise and unambiguous syntax that allows developers to predict exactly how their contracts will be executed
-- **Security**: The Clarity language allows users to supply their own conditions for transactions that ensure that a contract may never unexpectedly transfer a token owned by a user
-- **No compiling**: Contracts written in Clarity are broadcasted on the blockchain exactly as they are written by developers. This ensures that the code developers wrote, analyzed, and tested, is exactly what gets executed
+- **Predictability**: The Clarity language uses precise and unambiguous syntax that allows developers to predict exactly how their contracts will be executed.
+- **Security**: The Clarity language allows users to supply their own conditions for transactions that ensure that a contract may never unexpectedly transfer a token owned by a user.
+- **No compiler**: Contracts written in Clarity are broadcasted on the blockchain exactly as they are written by developers. This ensures that the code developers wrote, analyzed, and tested, is exactly what gets executed.
```

Comment:
maybe worth aligning on the convention: I never end list items with `.`

Don't really have a preference but I'm certain we have inconsistencies in the docs

Reply:
Gotcha. I tend to prefer adding the period when the line items are full sentences, and leaving it out when they are not